sklearn_crfsuite/estimator.py

```diff
@@ -365,11 +366,11 @@ def predict_single(self, xseq):
 
         Returns
         -------
-        y : list of strings
+        y : np.array of strings
             predicted labels
 
         """
-        return self.tagger_.tag(xseq)
+        return np.array(self.tagger_.tag(xseq), dtype=object)
```

Comment:
Initially I thought we shouldn't change `predict_single` and `predict_marginals_single`, instead we could leave them returning lists and only use numpy arrays as an outer container for methods which return multiple predictions. But the way it's currently implemented also works, except that the docstrings are not correct, e.g. `np.array of lists of strings` would be what I suggest, but now we implemented `np.array of np.array of strings`.